src/lib/modules/platformenvironment/interfaces/PlatformEnvironment.interface.ts

```diff
@@ -0,0 +1,4 @@
+export interface PlatformEnvironment {
+  readonly userId: number
+  readonly platformName: string
```

Comment:
[developer note] you can use enums instead of string. Ignore it until we add new environment :)